fixtures/env.test-killswitch

```diff
@@ -0,0 +1,18 @@
+CONFIG_PATH=../../fixtures/killswitch_config.json
```

Comment:
I was under the impression that this would *not* run in an agent-like environment and that `*URL` would be preferred over `*PATH` as it would be pointed at the last known good config in the config repo. Can add for the sake of completeness / flexibly though